src/openforms/scss/components/builder/_builder.scss

```diff
@@ -79,13 +79,6 @@ div.flatpickr-calendar.open
 // bootstrap competing with django admin form styling...
 .react-form-create {
   :not([ref="editForm"]) {
-
-    .form-row {
-      &, * {
-        box-sizing: initial;
-      }
-    }
-
```

Comment:
This causes misalignment of the error messages:

![image](https://user-images.githubusercontent.com/5518550/141815476-715aaa69-763b-41fe-8ebd-26c097bbc9f0.png)